internal/provisioner/kops_provisioner_cluster.go

```diff
@@ -419,6 +453,20 @@ func (provisioner *KopsProvisioner) UpgradeCluster(cluster *model.Cluster) error
 		return errors.Wrap(err, "failed to update kops instance group AMIs")
 	}
 
+	// TODO: read from config file
+	// TODO: check if those configs are already or remove this when we update all clusters
+	logger.Info("Updating kubelet options")
+	setValue := "spec.kubelet.authenticationTokenWebhook=true"
+	err = kops.SetCluster(kopsMetadata.Name, setValue)
+	if err != nil {
+		return err
+	}
+	setValue = "spec.kubelet.authorizationMode=Webhook"
+	err = kops.SetCluster(kopsMetadata.Name, setValue)
+	if err != nil {
+		return err
+	}
```

Comment:
Should this part be done in the `Provision` process? It seems like something we don't need to actually upgrade the cluster for.